project/Http4sPlugin.scala

```diff
@@ -37,22 +35,15 @@ object Http4sPlugin extends AutoPlugin {
     // Many steps only run on one build. We distinguish the primary build from
     // secondary builds by the Travis build number.
     isTravisBuild := sys.env.get("TRAVIS").isDefined,
-    http4sPrimary := sys.env.get("TRAVIS_JOB_NUMBER").fold(true)(_.endsWith(".1")),
     
     // Publishing to gh-pages and sonatype only done from select branches and
     // never from pull requests.
     http4sPublish := {
       sys.env.get("TRAVIS").contains("true") &&
         sys.env.get("TRAVIS_PULL_REQUEST").contains("false") &&
         sys.env.get("TRAVIS_REPO_SLUG").contains("http4s/http4s") &&
-        sys.env.get("TRAVIS_JDK_VERSION").contains("oraclejdk8") &&
-        (sys.env.get("TRAVIS_BRANCH") match {
-           case Some("master") => true
-           case Some(branch) if branch.startsWith("series/") => true
-           case _ => false
-         })
```

Comment:
The old TRAVIS_BRANCH check is covered by .travis.yml. The old oraclejdk8 reference is why #2570 failed to publish, and is also adequately covered by .travis.yml.